.github/workflows/ci.yml

```diff
@@ -31,12 +31,6 @@ jobs:
     - run: yarn install --frozen-lockfile --network-timeout 1000000
     - run: yarn build-all
 
-    - name: Define ToT chrome path
```

Comment:
I _think_ these can be removed because the tests can then use the stable Chrome already installed on the test runners, but we'll see how that goes :)

Reply:
But what's the benefit to using stable chrome vs ToT for these tests? presumably we used ToT for a reason?

Reply:
ToT lets us catch things early. We'd be losing that with this change. Don't recall if that has ever happened for unit/basics, but def. has for smoke.

Reply:
> Don't recall if that has ever happened for unit/basics

well ToT has only been used since April 15th, before that it was using the Chromium shipped with puppeteer.

Making tests simpler to run is almost always better, you end up with fewer hidden assumptions (like `CHROME_PATH` will always be set), but I can leave this in and add a comment it's specifically to catch ToT behavior